draft-ietf-taps-interface.md

```diff
@@ -1378,19 +1504,19 @@ does provide the following guarantees about the ordering of operations:
 
 # Transport Properties {#transport-props}
 
-Having discussed Transport Properties in {{connection-props}} and listed a
-number of Message Properties ("Ordered", "Idempotent" etc.) in {{message-props}},
+Having discussed Selection Properties in {{connection-props}}, Connection Properties {{connection-props}} and,  Message Properties ({{message-props}}),
```

Comment:
Two nits here: 
1. The first link should point to {{selection-props}}.
2. The comma should be before "and", not after.

Reply:
Thanks for this findings - fixed